### ISSUE TITLE
Syndicate to Medium

### ISSUE THREAD

**Issue by jhabdas** [mmistakes/made-mistakes-jekyll#50]
Though you might find this valuable: https://www.aaron-gustafson.com/notebook/crossposting-to-medium-from-jekyll/

**Comment by mmistakes** [comment]
Thanks for sharing.

**Comment by jhabdas** [comment]
Np

**Comment by jhabdas** [comment]
I can't help but keep thinking moving content to Medium will only serve to canibalize my readership. What are your thoughts, Michael?

Also, have you read "The Web We Have to Save"?

**Comment by mmistakes** [comment]
@jhabdas I think that's a valid concern. I haven't read that particular article but have seen similar sentiments expressed.

The whole idea of cross-posting content to Medium and friends seemed to spawn a few articles months ago. I don't have any links handy but I do remember reading a few for and against pieces about it. A pro "cross-posting" inspired me to test the waters then by taking one of my newer Jekyll articles and copying it on Medium.

Hard to say if that had any affect on my readership as it was just one article. I don't really notice any significant referral traffic nor have my metrics increased or decreased from it. Could be that it was only one article. Could be the type of article. Could be the subject matter. Could be anything really.

I know I've seen quite a few "Jekyllers" who integrate syndicated content into their sites. Perhaps it would be worth asking for feedback on talk.jekyllrb.com. You'll probably get better data from those who do the cross-posting thing more. All I can offer are opinions.

**Comment by jhabdas** [comment]
I put some thought into it and there are problems I see with cross-posting:

1. Duplication of content. Unlike RSS, I don't view cross-posting as a form of syndication. I view it as a form of duplication, which is unhealthy for the Web. Two glaring problems I saw after looking further into the Jekyll cross-posting Gem include a documentation snafu which caused canonical backlinks to become busted in a pretty gnarly way and inability for the Gem to perform revisions on Medium.
2. Loss of authority. When owners of content cross-post to Medium they are effectively giving up the value of their authoritativeness and handing it over to a social network. While I understand Medium allows individuals control over the copyrights of their content, the domains from which the content comes from become devalued by the human gaze.
3. Lackluster user experience. Much of the content written on Jekyll leverages certain traits and behaviors embedded in the code which Medium simply does not support. For example, notices from Minimal Mistakes will now appear properly upon cross-posting.
4. Lack of individualism. One of the things that makes the Web beautiful is the creativity which goes into it. Though Medium allows some degree of white-label branding all of the pages look the same, which in my mind impedes creative expression.

It wasn't until I thought more about your response, Michael, that I was able to think through this, so thanks for that.
